Dynatrace/dynatrace-operator
# Introduce version v1beta2 for Dynakube

## Description

With this a new version `v1beta2` of the Dynakube is introduced.

Noteworthy changes:
- a new field `metaDataEnrichment` was added to the .spec of the Dynakube.  it has an enabled field (enabled by default) and a namespace selector.
- a new field `dynatraceApiRequestsThreshold` was added to the .spec of the Dynakube.
- a new field `secCompProfile` was added to relevant oneAgent sections: `hostMonitoring`, `classicFullStack`, `cloudNativeFullstack`

Please have a look at the tags if they are correct and also check my wording please.

Corresponding JIRA ticket: [K8S-9331](https://dt-rnd.atlassian.net/browse/K8S-9331)

## How can this be tested?

Not sure how this could be tested. Maybe run `make manifests` and check the dynakube CRD.

## Checklist
